### PR TITLE
Fixed line spacing in command example

### PR DESCRIPTION
**Pull request by ghost**
Related PRs: https://github.com/openshift/openshift-docs/pull/6081 and https://github.com/openshift/openshift-docs/pull/7394

### CODE REVIEW

File: dev_guide/pod_autoscaling.adoc [hunk 1]
```diff
@@ -92,8 +92,8 @@ are given default values from the {product-title} server.
 For example:
 
 ----
-$ oc autoscale dc/frontend --min 1 --max 10 /
-  --cpu-percent=80 deploymentconfig "frontend" autoscaled
+$ oc autoscale dc/frontend --min 1 --max 10 --cpu-percent=80
```
Comment (ghyde) [inline dev_guide/pod_autoscaling.adoc]: You have trailing white space here.

**Review by ghyde — CHANGES_REQUESTED** [review]

**Comment by tmorriso-rh** [comment]
@ghyde Does this fix the spacing issue? Thanks.

**Comment by ghyde** [comment]
@tmorriso-rh LGTM

**Pull request merged**